src/referencePackages/src/system.text.json/6.0.0/lib/net6.0/System.Text.Json.cs

```diff
@@ -7,7 +7,7 @@
 [assembly: System.Runtime.CompilerServices.CompilationRelaxations(8)]
 [assembly: System.Runtime.CompilerServices.RuntimeCompatibility(WrapNonExceptionThrows = true)]
 [assembly: System.Diagnostics.Debuggable(System.Diagnostics.DebuggableAttribute.DebuggingModes.IgnoreSymbolStoreSequencePoints)]
-// Commenting out because JsonSerializerOptionsUpdateHandler is an internal type
+// Commenting out because JsonSerializerOptionsUpdateHandler is an internal type (https://github.com/dotnet/sdk/issues/31098)
```

Comment:
The assembly attribute `[assembly: System.Reflection.Metadata.MetadataUpdateHandler(typeof(JsonSerializerOptionsUpdateHandler))]` shouldn't be generated anymore with the GenAPI used in main. Can you please remove the comments?

Reply:
No, I mistakenly added it back. Regenerating it removes this content. Updated.